app/src/main/java/de/tum/in/tumcampusapp/database/viewmodel/CafeteriaViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package de.tum.`in`.tumcampusapp.database.viewmodel
+
+import android.arch.lifecycle.ViewModel
+import android.location.Location
+import de.tum.`in`.tumcampusapp.database.repository.LocalRepository
+import de.tum.`in`.tumcampusapp.database.repository.RemoteRepository
+import de.tum.`in`.tumcampusapp.models.cafeteria.Cafeteria
+import io.reactivex.Flowable
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+
+class CafeteriaViewModel(val localLocalRepository: LocalRepository, val remoteRepository: RemoteRepository,
+                         val compositeDisposable: CompositeDisposable) : ViewModel() {
+
+
+    fun getAllCafeteria(location: Location): Flowable<List<Cafeteria>> {
+        return localLocalRepository.getAllCafeterias()
```

Comment:
when you have a top level return statement, you can replace it with 
```kotlin
fun getAllCafeteria(location: Location) = 
    localLocalRepository.getAllCafeterias()
// etc
```

Reply:
yeah i already did that :)